docs/reference/indices/field-usage-stats.asciidoc

```diff
@@ -128,7 +128,8 @@ The API returns the following response:
                         "doc_values" : 1,
                         "points" : 0,
                         "norms" : 1,
-                        "term_vectors" : 0
+                        "term_vectors" : 0,
+                        "vectors" : 0
```

Comment:
Maybe we could call this "knn_vectors". This matches the naming in Lucene (`KnnVectorField`, `KnnVectorsFormat`). In other places, we've received feedback that "vector" is too ambiguous in the context of Elasticsearch/ Lucene.

Reply:
6e42856 👍